### PR TITLE
Chore gh actions 184638363

### PR DESCRIPTION
**Pull request by divinrkz**
#### What does this PR do?

This PR serves to integrate Node.js CI/CD github actions workflows.

#### Description of  the task to be completed?

Has the following functionalities:

-  Setup repo CI steps.
- Setup repo CD step.
- Setup readme status badge.

#### How should this be manually tested?

1. After making a PR or merging the `develop` branch, you can check the [Actions](https://github.com/atlp-rwanda/rca-barefoot-team6-bn/actions) page on the github web to see how the workflows are running. 

2. You can also check the [readme](https://github.com/atlp-rwanda/rca-barefoot-team6-bn/blob/develop/README.md) file to see a realtime status for the jobs. 

#### Any background context you want to provide?

The jobs are currently passing since they haven't been any codes written, no tests implemented, and no deployment environments configured yet. Once, these are ready, the workflows will be evident.

#### What are the relevant pivotal tracker stories?

[#184638363](https://www.pivotaltracker.com/story/show/184638363)

#### Screenshots (if appropriate)

---

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Deploy
```
Comment (Nkbtemmy) [inline .github/workflows/cd.yml]: this is not a Deploy job; please look at the end of your workflow and put here the correct name here

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: CI
```
Comment (Nkbtemmy) [inline .github/workflows/ci.yml]: Also, this is `test CI` not just CI only

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]
Please remember to add also GitHub action for the pull request template

**Review by didiermun — APPROVED** [review]
Looks good to me

File: .github/workflows/cd.yml (hunk 1 above)
Comment (divinrkz) [reply .github/workflows/cd.yml]: @Nkbtemmy, I wrote this for the deploy job, but we don't have any deployment environment ready.

**Comment by divinrkz** [comment]
Hey @Nkbtemmy , I made changes to the PR.

**Pull request merged**